### ISSUE TITLE
[BUG] Same example as in #383, but throws an error with engine-seed.

### ISSUE THREAD

**Issue by trusktr** [Famous/engine#385]
I've taken the working sample that I've shown in #383 and tried to run it in engine-seed (off of commit [d317ce45cb](https://github.com/Famous/engine-seed/tree/d317ce45cba536dc5428e8e5c4355632468842e7)). The only change I've made to the source is change the `famous.*.*` dependencies to `require('famous/*/*')` format. Now instead of the code running like you'll see if you paste the example from #383 into a famous.org editor, you'll see this error:

```
Uncaught Error: This node does not have access to a transform component
```

Here's a link to the branch of my engine-seed fork where this error happens: https://github.com/trusktr/engine-seed/tree/position-bug

If you change line 116 from

```js
                    let cardNode = scene.addChild(new BetterNode)
```

to

```js
                    let cardNode = scene.addChild(new Node)
```

You'll notice that not using the extended Node class results in the error going away! (I was lacking a name for the extended Node so, BetterNode. :laughing:)

**Comment by trusktr** [comment]
Made fix: #386

**Comment by jd-carroll** [comment]
@trusktr what version of the `engine` were you running with `famous-seed`?

**Comment by jd-carroll** [comment]
@trusktr I just tried the sample above from your repo and was not able to reproduce the issue.  (I did not see an exception in the console.)

Can you check your `node_modules` to see what version of the `engine` was resolved?

**Comment by trusktr** [comment]
@jd-carroll I missed your reply to this, but it's already resolved with that change from INIT_DEFAULT_COMPONENTS to NO_DEFAULT_COMPONENTS. 

Engine resolved to 0.6.2, which doesn't have that change yet. I can reproduce simply by `npm install; npm run dev`, on the `position-bug` branch. The result looks like this:

<img width="789" alt="screen shot 2015-07-04 at 10 06 19 am" src="https://cloud.githubusercontent.com/assets/297678/8508799/5cf5d27a-2234-11e5-9fa7-f6947ae9a410.png">

**Comment by trusktr** [comment]
Fixed by #384.
